### ISSUE TITLE
Tutorial text on "Overview" 

### ISSUE THREAD

**Issue by Kisgus** [trustlines-network/feedback#36]
### Basic information:

* OS (Android / iOS / both):  iOS newest
* Version: 

### Issue description:

1) When starting the app, you are pre-selected to the "ETH" which still has the below outdated tutorial text

![img_3223](https://user-images.githubusercontent.com/18598519/37336106-aeb3f44c-26b0-11e8-80c5-287e6a2a5d8a.png)

**When switching to forexample EUR**

2) The tutorial text needs to be fixed to be within the "Overview" screen so no scrolling is required

![img_3222](https://user-images.githubusercontent.com/18598519/37336153-d14deaee-26b0-11e8-8f0d-4ddb73e4915f.png)

3) Add clickable red text underneath the QR / Link options "Already have ETH?". Clicking this text will open the "Receive ETH step 0" flow, to allow the advanced user to send ETH the trustlines wallet.

In other words:

![36800974-19d83d1e-1cb1-11e8-9926-27354e2f2b26](https://user-images.githubusercontent.com/18598519/37352879-7a5fb0ae-26de-11e8-933b-65249c6444b3.png)

Leading to the updated version (in progress) on 

![37092068-ac0c7fac-220a-11e8-8d6a-f90d8bdbd79d](https://user-images.githubusercontent.com/18598519/37352912-8a275c9e-26de-11e8-81cc-cdecbd947b20.png)

### Possible solution

- [x] Align tutorial text on "EUR" to be within the "Overview" page

- [x] Update tutorial text on "ETH" screen

- [x] Include red clickable "Already have ETH?" text below the QR code / Share link

- [x] Clicking this text leads to the "Receive ETH flow" step 0, reminding the new user to back up password.

**Comment by Kisgus** [comment]
Fixed
